paderbox/utils/pandas_utils.py

```diff
@@ -39,17 +42,37 @@ def py_query(
        a  b
     1  3  4
 
+    To access column names that aren't valid python identifiers (e.g. the name
+    contains a whitespace), you have to use the kwargs dictionary:
+    >>> df = pd.DataFrame([{'a b': 1, 'b': 2}, {'a b': 3, 'b': 4}])
+    >>> py_query(df, 'kwargs["a b"] == 1')
+       a b  b
+    0    1  2
+
+    When you need a package function, you have to specify it in the globals
+    dict. e.g.:
+    >>> import numpy as np
+    >>> df = pd.DataFrame([{'a': 1, 'b': 2}, {'a': 3, 'b': 4}])
+    >>> py_query(df, 'np.equal(a, 1)', globals={'np': np})
+       a  b
+    0  1  2
+
     Args:
-        data:
+        data: pandas.DataFrame
         query: str or list of str. If list of str the strings get join by a
-            logical and to be a str. For examples see doctest.
+            logical `and` to be a str. For examples see doctest.
              Note: Use index to get access to the index.
         use_pd_query:  Pandas query is much faster but limited
         allow_empty_result:
-        setup_code: Additional code which runs before the query conditions.
+        setup_code: legacy argument, Superseded by the globals argument.
+            Additional code which runs before the query conditions.
             You may use this for additional imports.
+        globals: Specify some global names. Useful for imports.
+        return_selection: Whether to return the selection of the data or the
+            selection indices.
```

Comment:
should this variable be called return_selection_indices?

Reply:
Hmm. I am not sure. In `pb_bss` selection are the indices.
But here it may be more clear to be more verbose.

Reply:
I am also in favor of renaming `return_selection`. Maybe `return_selection_output` might be an option to be consistent with `pb_bss`? But then the logic has to be swapped.

Reply:
Ok, I changed it to `return_selected_data`.